src/js/components/App.vue

```diff
@@ -41,74 +41,271 @@
-                        </draggable>
+                        </div>
+
+                        <div class="newtab__section_body">
+                            <draggable :list="section.items">
+                                <div
+                                    class="newtab__item"
+                                    v-for="(item, itemKey) in section.items"
+                                    :key="itemKey"
+                                >
+                                    <button
+                                        class="newtab__item_button newtab__item_button--delete"
+                                        @click="removeItem(sectionKey, itemKey)"
+                                    >
+                                        <i class="fas fa-trash-alt"></i>
+                                    </button>
+
+                                    <button
+                                        class="newtab__item_button newtab__item_button--edit"
+                                        @click="openItemModal(sectionKey, itemKey)"
+                                    >
+                                        <i class="fas fa-edit"></i>
+                                    </button>
+
+                                    <a class="newtab__item_body" :href="item.url">
+                                        <span v-if="item.image" class="newtab__item_image">
+                                            <img :src="item.image">
+                                        </span>
+
+                                        <span v-if="item.icon" class="newtab__item_icon">
+                                            <i class="fa-5x" :class="item.icon"></i>
+                                        </span>
+
+                                        <span v-if="item.title" v-text="item.title"></span>
+                                    </a>
+                                </div>
+                            </draggable>
+                        </div>
                     </div>
                 </div>
             </draggable>
 
+            <modal
+                name="birthdayMonthModal"
+                :width="435"
+                :height="400"
+            >
+                <div class="newtab__modal">
+                    <div class="newtab__modal_header">
+                        <h1 class="newtab__modal_title" v-text="'Birthdays of the Month'"></h1>
+
+                        <button class="newtab__modal_close" @click="closeBirthdaysMonthModal()">
+                            <i class="fas fa-times"></i>
+                        </button>
+                    </div>
+
+                    <div class="newtab__modal_body">
+                        <div
+                            v-for="(person, personKey) in birthdayMonthModal"
+                            :key="personKey"
+                            class="newtab__modal_field">
+                            {{ person.name + ' - ' + person.date }}
```

Comment:
I *personally* don't like keeping logic on the HTML part, so all concatenation, v-ifs validations and etc, I keep on separated computed properties. But again, it's my personal preference to not have any logic (even little) on the View part

Reply:
change to `v-text="person.name + ' - ' + person.date"` temporary